airqo-platform/AirQo-frontend
# Locate form results frontend

- What JIRA task is related to this PR?
The link to the JIRA board task is [https://airqoteam.atlassian.net/browse/LOC-10](JIRA CARD URL).
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies. Afterwards run `npm run local`
- Which end points should are ready for testing?
None
